src/build/validator.md

```diff
@@ -1,7 +1,32 @@
 ---
-title: Validator Setup
+title: Validator Setup version 2
 ---
 
+## Version 2 updates
+
+Find the full list of exact updates **20/09/2021**
+1. Add new section [How to separate your running nodes](/build/validator.html#how-to-separate-your-running-nodes)
+2. Update the parameter **PRIVATE_PEERS** in the step 6 within the section [Configure and start the Sentry node as a validator](/build/validator.html#configure-and-start-the-sentry-node-as-a-validator)
```

Comment:
is the expectation to have all version notes at the top of this page, or will they be moved to a separate one?

Reply:
For sure on a separate page. But, for now, let's keep it here to avoid confusion for validators